modules/common-attributes.adoc

```diff
@@ -49,7 +49,7 @@ endif::[]
 :osp: Red Hat OpenShift
 :olm-first: Operator Lifecycle Manager (OLM)
 :olm: OLM
-:rhacs-version: 3.73.4
+:rhacs-version: 3.73.5
```

Comment:
So it looks like the variable for `rhacs-version` is being used in places where we want the latest version of the patch to appear (for example, https://docs.openshift.com/acs/3.74/configuration/enable-offline-mode.html#topic-name_enable-offline-mode). So we do still need that rhacs-version variable for those, and it always to be updated to the latest version, because we want people to use that latest patch, not any in-between releases. Unless I'm missing another use case where we need a variable for each patch release - other than the release notes, which is a one-off use?

Reply:
oh okay, I might have misunderstood the purpose of this attribute. Thanks for clarifying.

Reply:
I wasn't sure either, no worries!